src/api/app/models/history_element/review_accepted.rb

```diff
@@ -5,7 +5,7 @@ def description
     end
 
     def user_action
-      'accepted review'
+      'accepted the request'
```

Comment:
In "normal" circumstances (not in all) accepting the request might happen if you accept the last review. But not always.

Reply:
I am not sure I get it, maybe there are corner cases I am not aware of.
My main puzzling part starts by having "the result of a review" as a type of itself. A review should be _asked_ or _given_, and the result of that action is _accepting|declining_ a request. Whenever I submit a review I accept or decline the request, not the review. Am I missing something?

Maybe, as an agreement, we could reword this into `submitted a review`?

Reply:
>  A review should be asked or given, and the result of that action is accepting|declining a request.

Exactly here lies your misunderstanding.  A review *is* asked for and the result of that action is accepting|declining **the review**.

Reply:
I reverted the change.

It's getting out of scope as it is more about the workflow and the logic behind it, that is probably a bit tricky. I will track this down in a separate issue/card.

Thank you so far